packages/gatsby-plugin-offline/src/gatsby-browser.js

```diff
@@ -23,6 +24,8 @@ exports.onServiceWorkerActive = ({
   // get all resource URLs
   const headerResources = [].slice
     .call(nodes)
+    // don't include preconnect/prefetch/prerender resources
+    .filter(node => !ignoredLinkRels.test(node.getAttribute(`rel`)))
```

Comment:
If we move to a whitelist we have to change this line into a correct condition

```suggestion
    .filter(node => whiteListLinkRels .test(node.getAttribute(`rel`)))
```